libs/api/domains/rights-portal/src/lib/rightsPortal.service.ts

```diff
@@ -90,18 +98,103 @@ export class RightsPortalService {
       if (!res) return null
       return {
         currentDentistName: res[0].name,
-        billHistory: res[1] as DentistBill[],
+        billHistory: res[1] as Bill[],
       }
     } catch (e) {
       return handle404(e)
     }
   }
 
-  async getHealthCenterHistory(
+  async getDentists(user: User): Promise<PaginatedDentistsResponse | null> {
+    try {
+      const res = await this.dentistApi
+        .withMiddleware(new AuthMiddleware(user as Auth))
+        .dentists({ contractType: '' })
+
+      if (!res || !res.dentists || !res.totalCount || !res.pageInfo) {
+        return null
+      }
+
+      const dentists: Array<Dentist> = res.dentists
+        .map((d) => {
+          if (!d.id) {
+            return null
+          }
+          return {
+            ...d,
+            id: d.id,
+            address: {
+              postalCode: d.postcode,
+              municipality: d.region,
+              streetAddress: d.address,
+            },
+          }
+        })
+        .filter(Boolean as unknown as ExcludesFalse)
```

Comment:
We're going to refactor this away soon. Instead of using `Boolean as unknown as ExcludesFalse`, try the `IsDefined`filter from the shared utils library

Reply:
No rush on this tho, just if you're refactoring anyway